trench/views/simplejwt.py

```diff
@@ -12,8 +10,8 @@ def handle_user_login(self, request, serializer, *args, **kwargs):
         token = RefreshToken.for_user(serializer.user)
         return Response(
             {
-                'refresh': text_type(token),
-                'access': text_type(token.access_token)
+                'refresh': token,
+                'access': token.access_token
```

Comment:
To maintain the functionality of text_type, I think these will need to be cast to str instead. That may address the failing tests about things not being serializable.